app/build.gradle

```diff
@@ -82,6 +82,13 @@ android {
         viewBinding = true
     }
 
+    buildFeatures {
+        compose true
+    }
+    composeOptions {
+        kotlinCompilerExtensionVersion "1.2.0-rc02"
```

Comment:
I would prefer to use a stable version of this.

Reply:
we are using kotlin version `1.6.21` in the project and hence we need this version
![image](https://github.com/openMF/mifos-mobile/assets/90026952/160f9495-692f-48d6-81b8-efc6a36314a3)

We can either continue with this or we would have to upgrade the kotlin version . Although upgrading the kotlin version may come with problem of its own

Reply:
Ah, okaay, let's use this version temporarily then.